src/http.ts

```diff
@@ -345,6 +346,19 @@ export class LemmyHttp {
     );
   }
 
+  /**
+   * Hide a community from public view.
+   *
+   * `HTTP.PUT /community/hide`
+   */
+  hideCommunity(form: HideCommunity) {
+    return this.#wrapper<HideCommunity, CommunityResponse>(
+      HttpType.Put,
+      "/community/hide",
+      form,
+    );
+  }
+
```

Comment:
If I understand correctly, a hidden community should only be visible to the instance it's local to?

Reply:
It hides all posts / comments from hidden communities from `Local` and `All`, and any community listings. So only admins or ppl who know the url can go to it and subscribe.

"Hidden from public" seems like the easiest way to denote that, but I can change that if necessary.